src/pages/PoolDetails/PoolDetailsLink.tsx

```diff
@@ -0,0 +1,165 @@
+import { Trans } from '@lingui/macro'
+import { ChainId } from '@uniswap/sdk-core'
+import { EtherscanLogo } from 'components/Icons/Etherscan'
+import { ExplorerIcon } from 'components/Icons/ExplorerIcon'
+import CurrencyLogo from 'components/Logo/CurrencyLogo'
+import Row from 'components/Row'
+import { useInfoExplorePageEnabled } from 'featureFlags/flags/infoExplore'
+import { chainIdToBackendName, getTokenDetailsURL } from 'graphql/data/util'
+import { Token } from 'graphql/thegraph/__generated__/types-and-hooks'
+import { useCurrency } from 'hooks/Tokens'
+import useCopyClipboard from 'hooks/useCopyClipboard'
+import { useCallback } from 'react'
+import { ChevronRight, Copy } from 'react-feather'
+import { useNavigate } from 'react-router-dom'
+import styled, { useTheme } from 'styled-components'
+import { BREAKPOINTS } from 'theme'
+import { ClickableStyle, EllipsisStyle, ExternalLink, ThemedText } from 'theme/components'
+import { shortenAddress } from 'utils'
+import { ExplorerDataType, getExplorerLink } from 'utils/getExplorerLink'
+
+import { DoubleCurrencyLogo } from './PoolDetailsHeader'
+import { DetailBubble, SmallDetailBubble } from './shared'
+
+const TokenName = styled(ThemedText.BodyPrimary)`
+  display: none;
+
+  @media (max-width: ${BREAKPOINTS.lg - 1}px) and (min-width: ${BREAKPOINTS.xs - 1}px) {
```

Comment:
question for my own understanding: why don't we use `@media` for all our other responsive css as well? why do we use `@media only screen` or `@media screen` when virtually all browsers support media queries + do we need to specify screen for responsive purposes?

Reply:
we currently don't enforce one particular standard over the other so I think it just comes to implementers style. This might be something we want to codify via linter however if we see a compelling reason to use one over the other